src/components/common/helpers/Helpers.tsx

```diff
@@ -394,21 +395,51 @@ export function deepEqual(configA: any, configB: any): boolean {
 
 export function useOnline() {
     const [online, setOnline] = useState(navigator.onLine)
+    const realTimeConnectivityAbortRef = useRef<AbortController | null>(null)
```

Comment:
```suggestion
    const realTimeConnectivityAbortRef = useRef<AbortController | null>(new AbortController())
```
And we can remove null check for current